File: Hash Table/1832. Check if the Sentence Is Pangram.py
class Solution:
    def checkIfPangram(self, sentence: str) -> bool:
        alpha = set() 
        
        for letter in sentence:
            if ord(letter) in alpha:
                continue
            else:
                alpha.add(ord(letter))
        
        if len(alpha) != 26:
            return False
        else:
            return True

File: Hash Table/1832. Check if the Sentence Is Pangram.c
bool checkIfPangram(char * sentence){
    int alpha[26] = {};
    int count = 0;
    while (*sentence != '\0'){
        int index = (int) *sentence++ - (int) 'a';
        if (alpha[index] != 1){
            alpha[index] = 1;
            count++;
        }
        else{
        }
    }
    if (count == 26){
        return true;
    }
    else{
        return false;
    }
}
